File: python/profiling/profiling-no-db.py
def light_loop():
    for i in range(0, 1000 * 1000):
        a = i * i

def heavy_loop():
    for i in range(0, 10 * 1000 * 1000):
        a = i * i

def mid_tier():
    light_loop();
    heavy_loop();

def top_tier():
    mid_tier();

top_tier();

File: js/profiling/profiling-no-db.js
function light_loop() {
  var a = 0;
  for (var i = 0; i < 10 * 1000 * 1000; i++) {
    a = i * i;
  }
}

function heavy_loop() {
  var a = 0;
  for (var i = 0; i < 100 * 1000 * 1000; i++) {
    a = i * i;
  }
}

function mid_tier() {
  light_loop();
  heavy_loop();
}

function top_tier() {
  mid_tier();
}


top_tier();

